homeassistant/components/hassio/issues.py

```diff
@@ -263,20 +271,10 @@ def add_issue(self, issue: Issue) -> None:
     async def add_issue_from_data(self, data: IssueDataType) -> None:
         """Add issue from data to list after getting latest suggestions."""
         try:
-            suggestions = (await self._client.get_suggestions_for_issue(data["uuid"]))[
-                ATTR_SUGGESTIONS
-            ]
-            self.add_issue(
-                Issue(
-                    uuid=data["uuid"],
-                    type_=data["type"],
-                    context=data["context"],
-                    reference=data["reference"],
-                    suggestions=[
-                        Suggestion.from_dict(suggestion) for suggestion in suggestions
-                    ],
-                )
-            )
+            data["suggestions"] = (
+                await self._client.get_suggestions_for_issue(data["uuid"])
+            )[ATTR_SUGGESTIONS]
+            self.add_issue(Issue.from_dict(data))
```

Comment:
Site note: We try to limit the try... except block to the lines that can raise the exception. I'd move this line below and return in the except block.